examples/docs_snippets/docs_snippets/concepts/ops_jobs_graphs/op_hooks_context.py

```diff
@@ -7,7 +7,7 @@
 
 @failure_hook
 def my_failure_hook(context: HookContext):
-    op_exception: BaseException = context.op_exception
+    op_exception: BaseException = context.op_exception  # type: ignore  # (possible none)
```

Comment:
I dont think having these in our docs is good - let figure out a different way

Reply:
They are stripped out of the docs, I added that functionality [here](https://github.com/dagster-io/dagster/pull/11335)

Reply:
ah cool